Makefile

```
@@ -105,7 +114,7 @@ typecheck-python:
 
 changeset:
 	@yarn changeset
-	@yarn prettier --write ".changeset/**/*.md"
+	@yarn prettier --write --cache --cache-strategy content ".changeset/**/*.md"
```

This will run in local dev, where we don't want caching. The number of files matched by this glob will never be more than a few, so this should always be super fast regardless.

```suggestion
	@yarn prettier --write ".changeset/**/*.md"
```